node/node_test.go

```diff
@@ -35,7 +33,7 @@ func TestRun(t *testing.T) {
 	cometListener, err := net.Listen("tcp", cometHTTPAddress)
 	require.NoError(t, err)
 	app := testapp.NewTest(t, chainID.String())
-	ethstatedb := state.NewDatabase(rawdb.NewMemoryDatabase())
+	ethstatedb := testutils.NewEthStateDB(t)
```

Comment:
Do we need this extra variable? I think we can place the `testutils.NewEthStateDB` call with the other similar calls on line 52.

Reply:
Do you mean doing this directly on line 52?
```
n := node.New(
//...other code
testutils.NewEthStateDB(t)
//...other code
)
```

Reply:
Yes.